lib/did/service.ex

```diff
@@ -0,0 +1,14 @@
+defmodule DID.Service do
+  use Grax.Schema
+
+  schema DID.Service do
+    # As defined by https://www.w3.org/TR/did-core/#service-properties
+
+    # TODO: this is not formally defined in TTL files, how to proceed?
+    # TODO: type = :string | list_of(:string)
+    # property :type, ???, type: list_of(:string), required: true
+
+    # TODO: type = :iri | :json | list_of(:iri | :json)
```

Comment:
@marcelotto some of the types in the specs are union types, with this one being the most extreme example I could find. Can we properly model this with Grax, or do we have to find a workaround?

Reply:
Union types are supported for links only: https://rdf-elixir.dev/grax/schemas.html#union-links
This does not exist for properties. We would only not specify a `:type` in such cases, and would then have to take care of the validations on our own.

Reply:
OK good to know thanks. For now, our main focus is on emitting documents anyway so we can definitely go with the `:any` (or no explicit) type.

For parsing documents it would be quite nice to have the power of union types. Do you think it would make sense to add this to Grax at some point?

Reply:
> Do you think it would make sense to add this to Grax at some point?

At some point definitely. But I don't have time for that now, unfortunately.

Reply:
Sure, but then at least we could leave some comments there to change it later.